### DISCUSSION TITLE
Left the Course

### DISCUSSION THREAD

**Discussion by guerrillian** [Courseplay/Courseplay_FS22#2619] [Q&A]
So I tried with 1 headland and the maximum possible and the Tractor presented the same error in this spot.

The problem happens in the outer headland too.

The tractor is in the line, I don't know what could be.

![image](https://github.com/Courseplay/Courseplay_FS22/assets/128705907/1ed1db26-5521-4c20-bbe9-204e7fe940d6)

**Comment by Tensuko** [comment]
Something looks super broken.
I can't see any waypoints on your course.
Logfile pls.

**Comment by guerrillian** [comment]
Hi!

Here it is:

Thanks

[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/12038957/log.txt)

**Comment by guerrillian** [comment]
I think the course waypoints are generated according to the custom field waypoints, I started to remove a lot of them to make the editing easier, maybe that's the problem

This course in the screenshot presented the same error, but then I re generate it and the route is working fine now, it is beneath my understanding hehe

![image](https://github.com/Courseplay/Courseplay_FS22/assets/128705907/810d1e22-ab3e-4614-ba10-164dd00eff2a)

**Comment by Tensuko** [comment]
So you edited your course?
All course edit is on your own risk!
Left the course means nothing more then the driver was more then 50m away from its target waypoint.
So somehow you have no waypoints there.
The line shows only the connectiong between the waypoints, the driver will never follow the line!

**Reply by guerrillian** [reply]
wow

That's clear now! Now I understood it.

Thank you so much!

**Reply by Tensuko** [reply]
No Problem.

**Comment by guerrillian** [comment]
I closed but I think it is better to mark it as solved and I can't do it.

**Reply by Tensuko** [reply]
Its okay :)
